data/domains.csv

```diff
@@ -0,0 +1,1351 @@
+Domain Name,Domain Type,Agency,City,State
```

Comment:
`data.domains.csv` is still versioned in the repository. One TODO is to get this out of it, and have the `data/update.py` script download `domains.csv` from a consistent URL before it begins the scanning process.